content/ember/v5/deprecate-template-action.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Action helper and modifier 
+until: 6.0.0
+since: 5.10.0
+---
+
+### Scenario: `action` is passed a string
+
+Before:
+```handlebars
+<button type="button" {{action "plusOne"}}>
+  Click Me
+</button>
+```
+
+After
+
+```handlebars
+<button type="button" {{on 'click' this.plusOne}}>
+  Click Me
+</button>
+```
+or, if `plusOne` is passed in as an argument 
+```handlebars
+<button type="button" {{on 'click' @plusOne}}>
+  Click Me
+</button>
+```
+
+If the `plusOne` action is in an actions object, it needs to move out:
```

Comment:
Is it also possible that the action isn't defined in the direct parent component, as it bubbled up through the DOM tree? In that case the function reference will need to be passed as a component parameter. Would be good to also mention here.

It's been a years since I've used this syntax so could be mistaken though.

Reply:
I think you're right -- I've updated the text to mention this.